packages/vaadin-grid/src/vaadin-grid.js

```diff
@@ -385,18 +385,40 @@ class GridElement extends ElementMixin(
       },
 
       /**
+       * **Deprecated**
+       *
+       * See [`allRowsVisible`](#/elements/vaadin-grid#property-allRowsVisible)
+       *
        * If true, the grid's height is defined by its rows.
        *
        * Effectively, this disables the grid's virtual scrolling so that all the rows are rendered in the DOM at once.
        * If the grid has a large number of items, using the feature is discouraged to avoid performance issues.
+       *
        * @attr {boolean} height-by-rows
        * @type {boolean}
+       * @deprecated since version 14.7
```

Comment:
maybe we should have an indication that this will be removed in v22 like: `@deprecated(since=14.7, forRemoval=22)`